src/modules/entity-history/stores/current-entity-history-date-store.js

```diff
@@ -1,14 +1,13 @@
 import nuclearJS from 'nuclear-js';
-import dateToStr from '../../../util/date-to-str';
 import actionTypes from '../action-types';
 
 const { Store } = nuclearJS;
 
 const INSTANCE = new Store({
   getInitialState() {
-    const yesterday = new Date();
-    yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
I'll add this line back for this PR since it's a separate change, but this is something that didn't make much sense to me. Currently the Logbook defaults to showing all events from midnight today up until now, and the history page shows history for all of yesterday. I'd expect them both to behave the same.